src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -78,24 +78,32 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 			os.Exit(consts.FAILURE_EXIT_CODE)
 		}
 
-		scriptBuilder.WriteString("echo \"Removing existing modules directory...\"\n")
+		scriptBuilder.WriteString("echo \"Removing existing modules directory from root...\"\n")
 		scriptBuilder.WriteString("rm -fr " + targetNodeModulesDir + "\n")
 		scriptBuilder.WriteString("mkdir -p " + targetNodeModulesDir + "\n")
 		scriptBuilder.WriteString("echo Extracting modules...\n")
 		scriptBuilder.WriteString("$extractionCommand\n")
+		
 		// Some versions of node, in particular Node 4.8 and 6.2 according to our tests, do not find the node_modules
 		// folder at the root. To handle these versions, we also add /node_modules to the NODE_PATH directory.
 		scriptBuilder.WriteString("export NODE_PATH=\"" + targetNodeModulesDir + "\":$NODE_PATH\n")
 		// NPM adds the current directory's node_modules/.bin folder to PATH before it runs, so commands in
 		// "npm start" can files there. Since we move node_modules, we have to add it to the path ourselves.
-		scriptBuilder.WriteString("export PATH=" + targetNodeModulesDir + "/.bin:$PATH\n")
+		scriptBuilder.WriteString("export PATH=." + targetNodeModulesDir + "/.bin:$PATH\n")
```

Comment:
We probably do not need this code anymore? since based on the comment above by default `./node_modules/bin` is added to path